yaml/converter/legacy/internal/config.go

```diff
@@ -65,12 +69,53 @@ func Convert(d []byte) ([]byte, error) {
 			toContainer(container),
 		)
 	}
+
 	pipeline.Volumes = toVolumes(from)
 	pipeline.Trigger.Branch.Include = from.Branches.Include
 	pipeline.Trigger.Branch.Exclude = from.Branches.Exclude
 
-	manifest := &droneyaml.Manifest{}
-	manifest.Resources = append(manifest.Resources, pipeline)
+	if len(from.Matrix.Include) > 0 {
+		for _, environ := range from.Matrix.Include {
+			name := []string{"default"}
+
+			for _, value := range environ {
```

Comment:
This loop can run with a random order, maybe I need to write a small helper function that sorts this map by keyname? Got a better idea? maybe just some random checksum based on the map values?